server/game/cards/characters/04/robbstark.js

```diff
@@ -0,0 +1,58 @@
+const DrawCard = require('../../../drawcard.js');
+
+class RobbStark extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => (
+                this.game.currentChallenge &&
+                this.game.currentChallenge.challengeType === 'military'
+            ),
+            match: this,
+            effect: ability.effects.modifyStrength(this.numberOfLoyalChars())
+        });
+
+        this.action({
+            title: 'Initiate ability',
```

Comment:
Can we change this to something like 'Stand and remove a character from the challenge' - that reflects what the ability is and isn't too long